lib/sparse/src/common/sparse_vector.rs

```diff
@@ -42,6 +42,21 @@ impl SparseVector {
         }
         score
     }
+
+    /// Check if this vector overlaps with another vector.
+    /// Warning: Expects both vectors to be sorted by indices.
```

Comment:
We can remove this warning because we validate this condition in `SparseVector` constructor